src/import-service/serverless.yml

```diff
@@ -0,0 +1,42 @@
+service: import-service
+
+frameworkVersion: '2'
+
+provider:
+  name: aws
+  runtime: nodejs12.x
+  region: eu-west-1
+  apiGateway:
+    shouldStartNameWithService: true
+  iamRoleStatements:
+    - Effect: "Allow"
+      Action: "s3:ListBucket"
+      Resource: "arn:aws:s3:::nodejs-aws-static"
+    - Effect: "Allow"
+      Action: "s3:*"
+      Resource: "arn:aws:s3:::nodejs-aws-static/*"
+
+functions:
+  importProductsFile:
+    handler: handler.importProductsFile
+    events:
+      - http:
+          path: import
+          method: get
+          cors: true
+          request:
+            parameters:
+            querystrings:
```

Comment:
you're missing 1 level of indentation here

Reply:
Fixed this, thank you